src/main/java/seedu/address/model/person/Person.java

```diff
@@ -207,8 +207,11 @@ public boolean hasValidTelegram() {
     /**
      * Returns true if the person has same primary and secondary email.
      */
-    public boolean hasSameEmail(Email secondaryEmail) {
-        return email.equals(secondaryEmail);
+    public boolean hasSameEmail() {
```

Comment:
Wondering if `hasRepeatedEmail` would be a more descriptive name?

Reply:
agreed! fixed in 4aff76d